Beam/resources/assets/js/components/IotaScrolledToHere.vue

```diff
@@ -0,0 +1,121 @@
+<style lang="scss">
+.ri-scrolled-to-here {
+  font-family: sans-serif;
+  font-weight: 400;
+  font-size: 13px;
+  color: white;
+  background-color: #3974d1;
+  position: fixed;
+  bottom: 20px;
+  left: 0;
+  padding: 10px 20px;
+  display: flex;
+  align-items: center;
+  z-index: 9999999;
+  &__mouse-icon {
+    height: 20px;
+    margin-right: 20px;
+  }
+  &__touch-icon {
+    height: 20px;
+    margin-right: 18px;
+    margin-left: -3px;
+  }
+}
+</style>
+
+<template>
+  <div class="ri-scrolled-to-here">
+    <svg
```

Comment:
What's this SVG for? Can't we load it from some file? (I don't see the component altering the svg in any way). Again, just by reading it I can't tell what it is doing.

Reply:
It's for the little mouse / touch icon in the bottom left corner.
I've extracted it to simple component which takes device prop and return correct icon.